src/Sulu/Bundle/ContentBundle/Controller/NodeController.php

```diff
@@ -961,11 +970,6 @@ private function persistDocument(Request $request, $type, $document, $language)
     {
         $data = $request->request->all();
 
-        $workflowStage = $this->getRequestParameter($request, 'state');
-        if (null !== $workflowStage) {
-            $data['workflowStage'] = $workflowStage;
-        }
```

Comment:
The state is not set by the request anymore. This happens now automatically by the `WorkflowStageSubscriber`, based on if a page is only saved or also published.